nixpkgs-overlays/hack-get/default.nix

```diff
@@ -48,5 +48,5 @@ self:
       };
 
   # Make an attribute set of source derivations for a directory containing thunks:
-  thunkSet = dir: lib.mapAttrs (name: _: self.hackGet (dir + "/${name}")) (builtins.readDir dir);
+  thunkSet = dir: lib.mapAttrs (name: _: self.hackGet (dir + "/${name}")) (lib.filterAttrs (_: type: type != "regular") (builtins.readDir dir));
```

Comment:
```suggestion
  thunkSet = dir: lib.mapAttrs (name: _: self.hackGet (dir + "/${name}")) (lib.filterAttrs (_: type: type == "directory" || type == "symlink") (builtins.readDir dir));
```

Reply:
Thoughts @Ericson2314 ?

Reply:
Can we somehow canonicalize the path and then only accept "directory" ?